### ISSUE TITLE
Add a sent timestamp to received messages.

### ISSUE THREAD

**Issue by Goldflame09** [klinker-apps/messenger-issues#146]
If in a dead spot and someone sends me a text, when I have service again and receive said text, I would like to be able to see in the details of the message, when time it was sent to me and what time I received it.

**Comment by klinker24** [comment]
I will plan to add it to the future, but this isn't going to be a major priority for the app yet, sorry

**Comment by anssisaari** [comment]
This issue has been around as long as Android. I hear Google fixed it in Hangouts though.

There's a separate app that gets the SMS sent time and appends it to messages (https://play.google.com/store/apps/details?id=at.zweng.smssenttimefix) but that doesn't work since Android 4.4. But still, they demonstrated the info can be had by an application so it should be doable.

In fact, apparently it was a one line fix to the old SmsReceiverService.java as shown here: https://code.google.com/p/android/issues/detail?id=2989#c87

**Comment by klinker24** [comment]
Yes, it is easy to get the actual time for messages now, that isn't a holdup.

**Comment by anssisaari** [comment]
Well, if it's easy then why not just do it? I don't understand this attitude at all. Is it so boring or what?

**Comment by klinker24** [comment]
Umm look on this page, I have plenty of things to do and need to prioritize. Just because it is easy doesn't mean it is trivial or quick.

**Comment by lahceneamine** [comment]
Hello Dev, first off all  I would like to thank you for this amazing app.
But I'm afraid that I'm going back to another app until this feature of getting both sent and recieve times (incoming and outgoing).
I really like your app, so please, it has been there since November, could you please do it ?

**Comment by klinker24** [comment]
Yes, so have half the other issues here. 

If it were a quick change, I would have done it. It will take time, so it will need prioritized, and it is not as high a priority as other things for me, since such a small subset of users even notice.

**Comment by lahceneamine** [comment]
Sad enough though.
Good luck then :)

**Comment by BlueMeany** [comment]
Would like this feature too. Its a shame that I have to check timestamp from within one of your rivals app (textra) when everything else in your app is superior
